package.json

```diff
@@ -40,7 +40,8 @@
     "mustache": "^2.2.1",
     "mutationobserver-shim": "^0.3.1",
     "pbf": "^1.3.2",
-    "point-in-polygon": "1.0.0",
+    "point-geometry": "^0.0.0",
+    "point-in-polygon": "steve9164/point-in-polygon",
```

Comment:
Ok, thanks, that makes sense.  A fork is ok, we should just put it in TerriaJS.  But I think I agree with you; this thing is so simple and unlikely to change that just putting the one function right in our code doesn't really have any downsides that I can think of.